File: 2025-2/Clase27/SextoProyecto/SextoProyecto/src/Cuadrado.java
public class Cuadrado extends Figura{

    public Cuadrado(String id, double base, double altura) {
        super(id, base, altura);
    }

    @Override
    public void area(){
        System.out.println("Area del Cuadrado es: " + getBase()*getAltura());
    }
}

File: 2026-1/Clase20/Figuras/Bibliotecas/Cuadrado.cpp
//
// Created by erasmo on 6/2/26.
//

#include "Cuadrado.hpp"

#include <iostream>
#include <ostream>
using namespace std;

void Cuadrado::area() {
    cout<<"Area del Cuadrado"<<endl;
}

void Cuadrado::leer() {
    Figura::leer();
    cout<<"Leer Cuadrado"<<endl;
}
